src/lang/english.txt

```diff
@@ -2866,6 +2866,11 @@ STR_HOUSE_PICKER_CLASS_ZONE3                                    :Outer Suburbs
 STR_HOUSE_PICKER_CLASS_ZONE4                                    :Inner Suburbs
 STR_HOUSE_PICKER_CLASS_ZONE5                                    :Town centre
 
+STR_HOUSE_PICKER_PROTECT_TITLE                                  :Prevent upgrades
+STR_HOUSE_PICKER_PROTECT_TOOLTIP                                :Choose whether the next house placed will be protected from replacement as the town grows
```

Comment:
This suggest it auto-disables after placing a single house. Is that true? Or is it just: `Choose whether placed houses will be ...` ?

Reply:
No, it doesn't auto-disable. But changing the setting won't protect houses you've previously placed.

How about `Choose whether this house will be protected from replacement as the town grows`?